README.md

```diff
@@ -218,6 +218,7 @@ Total deals: 218
 
 |  | Name | Description | Discount Code & Terms |
 | -- | ---| ------ | ------ |
+|  💰 | [Rize.io Time Tracker](https://rize.io) | Rize is an AI-powered time tracker that improves your focus and builds better work habits. | 40% OFF with code **BLACKFRIDAY2024** |
```

Comment:
Hi @macgill4444 , please move the deal to the bottom of the table. Thanks

Reply:
Sounds good! Updated the PR now.